### ISSUE TITLE
Edits to /me messages do not appear, even though "(edited)" does

### ISSUE THREAD

**Issue by tayloj** [mattermost/mattermost-server#16986]
#### Summary

Editing a message generate by a `/me` command has no effect, but still shows as "(edited)".

#### Steps to reproduce

In a chat, post:

```
/me makes a typoh
```

Press up to edit, or use the edit menu.  The Edit Post dialog appears, with `*makes a typoh*` in the text entry pane.  Change the text to `*makes a typo*`.  Press Save.  The message "makes a typoh" still appears, but "(edited)" appears below.

Version Info:

* Mattermost Version: 5.29.1
* Database Schema Version: 5.29.0
* Database: postgres

* Build Hash: 35ce42d96c2066cf083cb5b5c059d98506ce3cd3
* EE Build Hash: none
* Webapp Build Hash: 69386f97d80a1c6cbb7fed422922a378a1c8ef71
* Build Date: Thu Dec 3 18:31:39 UTC 2020

#### Expected behavior

Expected saved edits to appear.

#### Observed behavior (that appears unintentional) 

Saved edits did not appear, but "(edited)" did.

#### Possible fixes

n/a

**Comment by amyblais** [comment]
This is fixed in v5.32 via https://mattermost.atlassian.net/browse/MM-30810,

**Comment by tayloj** [comment]
@amyblais Thanks!  When I posted this issue, I did try to follow the instruction in the template that said to check the Atlassian page, 

> 1. Confirm you’re filing a new issue. [Search existing tickets in Jira](https://mattermost.atlassian.net/issues/) to ensure that the ticket does not already exist.

I get a 500 for that link, though.  I can see the link you posted just fine:

https://mattermost.atlassian.net/browse/MM-30810

It looks like I  have to go to https://mattermost.atlassian.net/jira/software/c/projects/MM/issues/ to browse issues on the Atlassian page.  Should the issue template be updated?
